### PR TITLE
Support CLICKHOUSE_REPO_ENABLED env var

### PR DESCRIPTION
**Pull request by IvanIvanoff**
## Changes

Support `CLICKHOUSE_REPO_ENABLED` env var that is used in the local development process. Currently, if sanbase is started without a CH connection it just crashes as the CH Ecto Repo cannot be started. The `MarkExchanges` service also needs CH connection in its initiation process.

Now sanbase can be started as `CLICKHOUSE_REPO_ENABLED=false iex -S mix phx.server` and it won't crash.
This can be used while the current task does not make any CH queries - all the postgres related data. This also makes it easy to work on something without internet connection.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/application.ex [hunk 1]
```diff
@@ -173,7 +173,10 @@ defmodule Sanbase.Application do
       SanbaseWeb.Telemetry,
 
       # Start the Clickhouse Repo
-      start_in(Sanbase.ClickhouseRepo, [:prod, :dev]),
+      start_if(
+        fn -> start_in(Sanbase.ClickhouseRepo, [:prod, :dev]) end,
+        fn -> System.get_env("CLICKHOUSE_REPO_ENABLED", "true") |> String.to_existing_atom() end
```
Comment (tspenov) [inline lib/sanbase/application.ex:178]: Can we reuse `Sanbase.ClickhouseRepo.enabled?()` here ?

Comment (IvanIvanoff) [reply lib/sanbase/application.ex:178]: Actually I introduced it in order to be used here, but forgot to replace it.

Comment (IvanIvanoff) [reply lib/sanbase/application.ex]: One more thing: It looks weird to have part of the check in the first function and the second check in the second function. I've reworked it.

File: lib/sanbase/clickhouse/mark_exchanges.ex [hunk 2]
```diff
@@ -118,27 +118,30 @@ defmodule Sanbase.Clickhouse.MarkExchanges do
...
-          Process.send_after(self(), :set_state, 5_000)
-
-          Map.put(%{}, :exchange_wallets_set, MapSet.new())
-          |> Map.put(:updated_at, Timex.now())
-      end
+  defp fill_state() do
+    ch_repo_enabled? = Sanbase.ClickhouseRepo.enabled?()
+    env = Application.get_env(:sanbase, :env)
+
+    case ch_repo_enabled? and env != "test" do
```
Comment (tspenov) [inline lib/sanbase/clickhouse/mark_exchanges.ex:125]: we compare with atoms above -> `Application.get_env(:sanbase, :env) in [:dev, :prod]`.

Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/mark_exchanges.ex:125]: I accidentally merged the PR, fixing it here: https://github.com/santiment/sanbase2/pull/2472

**Pull request merged**